test/intl402/DurationFormat/constructor-option-read-order.js

```diff
@@ -0,0 +1,46 @@
+/// Copyright 2023 the V8 project authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2023 the V8 project authors. All rights reserved.
```

Reply:
#3979 recently fixed some triple leading slashes, so maybe there are some tools which don't handle this case correctly.